tests/components/minio/test_minio.py

```diff
@@ -80,29 +80,41 @@ def minio_client_event_fixture():
     await hass.services.async_call(
         DOMAIN,
         "put",
-        {"file_path": "/tmp/some_file", "key": "some_key", "bucket": "some_bucket"},
+        {
+            "file_path": "/tmp/some_file",
+            "key": "some_key",
+            "bucket": "some_bucket"
+        },
         blocking=True,
     )
-    assert minio_client.fput_object.call_args == call(
-        "some_bucket", "some_key", "/tmp/some_file"
-    )
+    assert minio_client.fput_object.call_args == call("some_bucket",
+                                                      "some_key",
+                                                      "/tmp/some_file")
     minio_client.reset_mock()
 
     await hass.services.async_call(
         DOMAIN,
         "get",
-        {"file_path": "/tmp/some_file", "key": "some_key", "bucket": "some_bucket"},
+        {
+            "file_path": "/tmp/some_file",
+            "key": "some_key",
+            "bucket": "some_bucket"
+        },
         blocking=True,
     )
-    assert minio_client.fget_object.call_args == call(
-        "some_bucket", "some_key", "/tmp/some_file"
-    )
+    assert minio_client.fget_object.call_args == call("some_bucket",
+                                                      "some_key",
+                                                      "/tmp/some_file")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Probable insecure usage of temp file/directory.](https://app.codacy.com/app/foreign-sub/home-assistant/pullRequest?prid=4639533)